stellar/operations.py

```diff
@@ -36,14 +36,9 @@ def copy_database(raw_conn, from_database, to_database):
     terminate_database_connections(raw_conn, from_database)
 
     if raw_conn.engine.dialect.name == 'postgresql':
-        raw_conn.execute(
-            '''
-                CREATE DATABASE "%s" WITH TEMPLATE "%s";
-            ''' %
-            (
-                to_database,
-                from_database
-            )
+        sqlalchemy_utils.functions.create_database(
+            '%s%s' % (raw_conn.engine.url, to_database),
+            template=from_database
```

Comment:
It's just that the project has reached a point where it works good enough so there hasn't really been any need for changes. I'm still using it every week so it's far from being abandoned.